### ISSUE TITLE
Create guide module

### ISSUE THREAD

**Issue by thecodewarrior** [TeamWizardry/LibrarianLib#152]
Likely called "Compendium". I'm torn on whether this should include the UI or should be a slightly more abstract setup. The abstract version would still have a lot of the logic that would back a UI, but the library user would be free to configure how the UI actually appears.

**Comment by SaadArdati** [comment]
Imo convenience is better. The last guide book was a flawless plug n play ana made things... ludicrously convenient.
If a mod wants something different, they can do it themselves, or even just so their own UI but using the liblib logic. They shouldn't be coupled at all.

**Comment by thecodewarrior** [comment]
So basically both? An abstract system with a default GUI implementation?

**Comment by SaadArdati** [comment]
Yes, the json system should be easily portable from. 1.12 since it's very decoupled and relies on Lang files.
